react/index.js

```diff
@@ -46,4 +48,5 @@ let connectStoreon = (...keys) => {
   })
 }
 
-module.exports = { useStoreon, StoreContext, connectStoreon }
+
+export { useStoreon, StoreContext, connectStoreon, contextOverride }
```

Comment:
Can you also revert exports?

Reply:
Patched 6bb782a4016d2c6ae0a56521c2be0b83ab189577